app/views/notes/new.html.haml

```diff
@@ -0,0 +1,3 @@
+.form-container 
+  = render partial: "notes/form", locals: { notable: @student }
```

Comment:
`@student` seems wrong, but it also does not look like you are using the local in the form partial

```suggestion
  = render partial: "notes/form"
```